src/seedu/addressbook/AddressBook.java

```diff
@@ -176,15 +162,11 @@
      */
     private static ArrayList<HashMap<PersonProperty, String>> latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
 
-    /**
-     * The path to the file used for storing person data.
-     */
+    /** The path to the file used for storing person data. */
     private static String storageFilePath;
     
-    /**
```

Comment:
encourage you to try some more meaningful code improvements than just reformatting the comments.